spec/tic_tac_toe_spec.rb

```diff
@@ -0,0 +1,67 @@
+require_relative '../lib/interface.rb'
+require_relative '../lib/board.rb'
+require_relative '../lib/players.rb'
+
+describe Interface do
+  let(:interface) { Interface.new }
+  let(:test_array) { %w[1 2 3 4 5 6 7 8 9] }
+  it 'displays the board' do
+    expect(interface.display_board(test_array).class).to eql(String)
+  end
+end
+
+describe Board do
+  let(:board_obj) { Board.new(%w[1 2 3 4 5 6 7 8 9]) }
+  it 'Changes the possition given for "X" ' do
+    expect(board_obj.swap_on_board(2, 5)).to eql(board_obj.board)
+  end
+end
+
+describe Board do
+  let(:board_obj) { Board.new(%w[1 2 3 4 5 6 7 8 9]) }
+  it 'Changes the possition given for "O" ' do
+    expect(board_obj.swap_on_board(5, 9)).to eql(board_obj.board)
+  end
+end
```

Comment:
## Negative scenario

- Specs cover positive and negative scenarios  (screenshot  from  Pathwright)
Example: there is a spec for correct usage of the method [e.g. it "creates object if all required attributes are passed"]
and also there is spec for the wrong usage of a method  [e.g. it "does not create the object if required attributes are missing"].

  - [ ] Thus, please add negative scenarios

Example:

```ruby
#positive scenario
 it 'Changes the position given for "O" ' do
#negative scenario
 it 'doesn\'t change the position given for "O" ' do
```